GoogleForCreators/web-stories-wp
# Update Stories WordPress admin menu

## Summary

Update menu by adding new dumby menu items that redirect.

## Relevant Technical Choices

I was unable to cleanly re-order Add new and all stories. This would require rebuilding the menu array by editting the global, something I don't want to do / can't be done cleanly. 

To add in these menu items, I added new dumby menu items that 301 to where they need to go. 301ing is a not a great solution, as it reloads the page and does't show a current menu status. But without more work on the dashboard, I don't see how this is possible. 

Settings menu item is not displayed to users unless they have `manage_options` capability. 

Experiments was moved to the last menu item. 

 
## To-do



## User-facing changes

<img width="165" alt="Screenshot 2020-09-30 at 12 51 16" src="https://user-images.githubusercontent.com/841956/94683195-63075e80-0326-11eb-8dac-8d6b80d7fa5d.png">

## Testing Instructions

1. Verify menu items are in expected order. (see screenshot above)
1. Verify menu items link to the correct pages.
1. Verify menu items are identical for users with author role and admins and work for both

---



Fixes #2750
